### DISCUSSION TITLE
Revert back to ‘hold to seek’

### DISCUSSION THREAD

**Discussion by SanderRepo** [PoomSmart/YTABConfig#97] [General]
Previously in the YouTube app, you had the option to hold anywhere in the video to seek in the video using the timeline. YouTube changed the hold to a speed option (2x). How do I change this back to the seek option?

Thanks in advance!

**Comment by HoughtonMobile** [comment]
Use spoof app version and then clear app cache through uYou before force closing the app and rebooting your device and restarting the app once again .. should be back to old UI / old navigations / old style app in general , but still keeps the important aspects from new YouTube , if that makes sense ... 

That's what worked for me at least

**Reply by SanderRepo** [reply]
Thanks! That did the trick
